### PR TITLE
Add logo to README

### PR DESCRIPTION
**Pull request by MicheleBertoli**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
-[![npm version](https://badge.fury.io/js/snapguidist.svg)](https://badge.fury.io/js/snapguidist) [![Build Status](https://travis-ci.org/styleguidist/snapguidist.svg?branch=master)](https://travis-ci.org/styleguidist/snapguidist)
+[![npm version](https://badge.fury.io/js/snapguidist.svg)](https://badge.fury.io/js/snapguidist)
+[![Build Status](https://travis-ci.org/styleguidist/snapguidist.svg?branch=master)](https://travis-ci.org/styleguidist/snapguidist)
 
-# Snapguidist
-Snapshot testing for [React Styleguidist](https://github.com/styleguidist/react-styleguidist).
+<img alt="Preview" src="logo.png" width="450px" height="166px" />
```
Comment (sapegin) [inline README.md]: Why not SVG and Markdown? ;-)

Comment (MicheleBertoli) [inline README.md]: The font of the logo is [Source Sans Pro](https://fonts.google.com/specimen/Source+Sans+Pro).

Comment (sapegin) [inline README.md]: 🐙

Comment (MicheleBertoli) [inline README.md]: :)

**Pull request merged**